src/components/comments/comment-area.tsx

```diff
@@ -56,6 +57,10 @@ export function CommentArea({
       window.location.hash.startsWith('#comment-')) ||
     highlightedCommentId !== undefined
 
+  const [showThreadChildren, setShowThreadChildren] = useState<string[]>(
```

Comment:
Why is a state necessary here?

Reply:
because some users really like the option to toggle the visibility of the comments in one thread

Reply:
I ask because `setShowThreadChildren` is nowhere used. Just a comment though -> no blocker :smile: